common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressOptions.java

```diff
@@ -38,6 +38,28 @@
     @Parameter(names = { "--sync" }, description = "Runs sync version of test")
     private boolean sync = false;
 
+    @Parameter(names = { "-s", "--size" }, description = "Size of payload (in bytes)")
```

Comment:
I was looking to keep all the options in one class so that users can find all available options in one place. Options that don't apply to a test can have appropriate error messages or just simply not use them.

Reply:
Also, if a test needs both size and count options, then a third class has to be created to include both options as the `PerfStressTest` ctor takes a single `PerfTestOptions` instance.

Reply:
Discussed offline, I slightly prefer separate classes to avoid having options which don't apply to all tests, but extra options aren't a big deal.